botocore/monitoring.py

```diff
@@ -459,12 +472,20 @@ def _get_access_key(self, request_headers):
         return auth_match.group('access_key')
 
     def _get_region(self, request_headers):
+        if not self._is_signed(request_headers):
```

Comment:
Why return no region if the request is unsigned?

Reply:
It is part of the specification. The region used in the event must come from the region used when signing.